src/components/FlowRunTimelineSubFlowRunDetails.vue

```diff
@@ -43,18 +40,50 @@
   const alternate = true
 
   const routes = useWorkspaceRoutes()
-
   const api = useWorkspaceApi()
+
   const flowRunSubscriptionArgs = computed<[string]>(() => [flowRunId.value])
   const flowRunSubscription = useSubscriptionWithDependencies(
     api.flowRuns.getFlowRun,
     flowRunSubscriptionArgs,
   )
 
   const subFlowRun = computed(() => flowRunSubscription.response)
```

Comment:
This subscription is more complex than it needs to be. You're just getting a single flow run right? 
```typescript
const { flowRunId } = toRefs(props)
const { flowRun: subFlowRun } = useFlowRun(flowRunId)
```

Reply:
ahhh, didn't know this existed

Reply:
Updated the task run details panel to use the composition as well.